contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer.py

```diff
@@ -72,17 +72,31 @@ def _execute_buildozer_script(self, command):
       Buildozer.execute_binary(command, address.spec, binary=self._executable)
 
   @classmethod
-  def execute_binary(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c'):
+  def execute_binary(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c', suppress_warnings=False):
     binary = binary if binary else BinaryUtil.Factory.create().select_binary('scripts/buildozer', version, 'buildozer')
+    Buildozer._execute_buildozer_command([binary, command, spec], suppress_warnings, output=False)
 
-    Buildozer._execute_buildozer_command([binary, command, spec])
+  @classmethod
+  def return_buildozer_output(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c', suppress_warnings=False):
+    binary = binary if binary else BinaryUtil.Factory.create().select_binary('scripts/buildozer', version, 'buildozer')
```

Comment:
Ah I see you probably cannot use `self._executable` because it is an instance variable and does not apply to a classmethod. 

The correct way is to make Buildozer a subsystem and `--version` being an option in that subsystem, so various tasks with buildozer subsystem can use the same version of buildozer. E.g. 
https://github.com/pantsbuild/pants/blob/0450732d8144e122282c1cfd256c12ae3e7abc2a/src/python/pants/ivy/bootstrapper.py#L65 then you can grab the global instance of buildozer subsystem and tell it to run certain commands.

This as it stands still needs some improvement. I understand as far as the class goes it ends tomorrow evening, but we will appreciate that you can follow up the effort if you cannot finish refactoring the subsystem into it by then. 

Function wise, I can consider this PR gets the point, but for production code, we have to gate keep a bit more :)

Reply:
Thank you for the advice and approach recommendation, I will try to implement it this way as soon as possible - and am looking forward to continuing development on pants after the class is over.